scripts/vmware-release

```diff
@@ -8,13 +8,81 @@ touch .make-vmware
 source ./scripts/version
 ./scripts/release
 
-ISO_NAME="rancheros-vmware.iso"
+ROOT_FS_NAME="rootfs-vmware.tar.gz"
+VMDK_NAME="rancheros.vmdk"
+
+if [ "$VMWARE_AUTOFORMAT" = 1 ]; then
+    ISO_NAME="rancheros-autoformat.iso"
+else
+    ISO_NAME="rancheros.iso"
+fi
+
+if [ "$VMWARE_APPEND" != "" ]; then
+    echo "--append ${VMWARE_APPEND}"
+    APPEND_PARAM="--append \"${VMWARE_APPEND}\""
+fi
+
+mkdir -p /tmp/dist/openstack/latest && mv ./dist/* /tmp/dist/
+
+# create rancheros-vmware.vmdk
+if [ -e /tmp/dist/vmdk.img ]; then
+    rm -rf /tmp/dist/vmdk.img
+fi
+
+cat > /tmp/dist/openstack/latest/user_data << EOF
+#!/bin/bash
+set -e
+
+trap "poweroff" EXIT
+
+mount -t 9p -o trans=virtio,version=9p2000.L config-2 /mnt
+
+touch log
+sleep 5
+openvt -s -- tail -f log &
+ros install \
+    -d /dev/vda \
+    ${APPEND_PARAM} \
+    -f \
+    --no-reboot >log 2>&1
+
+touch /mnt/success
+EOF
+
+rm -f /tmp/dist/success
 
-mkdir -p /tmp/dist && mv ./dist/* /tmp/dist/
 mkdir -p ./dist/vmware && mv /tmp/dist/* ./dist/vmware/
-mkdir -p ./dist/artifacts && cp ./dist/vmware/artifacts/rancheros.iso ./dist/artifacts/${ISO_NAME}
+mkdir -p ./dist/artifacts
+
+qemu-img create -f qcow2 ./dist/vmware/vmdk.img 8G
+kvm -curses \
+    -drive if=virtio,file=./dist/vmware/vmdk.img \
+    -cdrom ./dist/vmware/artifacts/rancheros.iso \
+    -m 2048 \
+    -fsdev local,id=conf,security_model=none,path=./dist/vmware \
+    -device virtio-9p-pci,fsdev=conf,mount_tag=config-2
+
+[ -f ./dist/vmware/success ]
+
+echo Converting ./dist/${VMDK_NAME}
+qemu-img convert -f qcow2 -O vmdk ./dist/vmware/vmdk.img ./dist/artifacts/${VMDK_NAME}
+
+cp ./dist/vmware/artifacts/rancheros.iso ./dist/artifacts/${ISO_NAME} && \
+    cp ./dist/vmware/artifacts/rootfs.tar.gz ./dist/artifacts/${ROOT_FS_NAME} && \
+    cp ./dist/vmware/artifacts/${INITRD} ./dist/artifacts/${INITRD}
+
+cat > ./dist/publish_gss_latest.sh << EOF
```

Comment:
Please refer to this: https://github.com/rancher/os/blob/master/scripts/release#L51-L70

Reply:
Already modified